5.analyze-data/utils/well_significance_testing.py

```diff
@@ -0,0 +1,177 @@
+from collections import defaultdict
+
+def store_comparisons(_comp_functions, _treatments, _con_probs, _treatment_probs, **_comp_names):
```

Comment:
```suggestion
def store_comparisons(_comp_functions, _treatments, _con_probs, _treatment_probs, **_comp_names) -> dict{list}:
```
Not sure of the proper way of putting a "dictionary of lists", but recommend including this in the function so that it is easy to identify what the function is returning before going into the docstring.

Reply:
Thanks for the suggestion